### PR TITLE
#166789956 Users Should Be Able To Create And Update Articles

### PR DESCRIPTION
**Pull request by kodek-sleuth**
#### What does this PR do?
Should enable user to create and update an article
#### Description of Task to be completed
- Setup actions for making call to API
#### How should this be manually tested?
```
Run the following commands in your terminal after cloning the project
1. cd ah-92explorers-frontend
2. git checkout ft-create-update-article-166789956
3. `npm i` to install all dependencies 
4. `npm run dev` and navigate to localhost:8080
```
#### What are the relevant pivotal tracker stories?
[#166789956](https://www.pivotaltracker.com/story/show/166789956)
#### Screenshots (if appropriate)
Screen Shot 2019-08-28 at 20 31 52
#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by Hervera — CHANGES_REQUESTED** [review]
The home page is blank. The recent functionality of logout doesn't work

**Review by Hervera — CHANGES_REQUESTED** [review]
You disabled too many `Eslint rules` in `Routes.js`. Even the important ones like `indent`, `extensions` and so on.

**Review by Hervera — DISMISSED** [review]
After creating an article, how can I see that article created and how can I update that Article? I do not know the slug to use

File: src/components/Routes.js [hunk 1]
```diff
@@ -1,25 +1,32 @@
-import React, { Component, Fragment } from 'react';
+/* eslint-disable react/jsx-filename-extension */
```
Comment (niyoceles) [inline src/components/Routes.js]: @kodek-sleuth Kindly you must remove these (Eslint disabled) and put them in the `.eslintrc file`

**Review by niyoceles — DISMISSED** [review]
To create an article is ok, but after creating an article, the button of Edit and delete must be available on that page

**Review by elemanhillary-zz — DISMISSED** [review]
PR title not sufficient and tests are not passing

**Review by Hervera — DISMISSED** [review]
The coverage has dropped

**Review by elemanhillary-zz — CHANGES_REQUESTED** [review]
ready for merge

**Review by elemanhillary-zz — APPROVED** [review]
ready for merge

**Review by niyoceles — APPROVED** [review]
Can be merged

**Review by IsaiahRn — APPROVED** [review]

**Pull request merged**